Widgets/Widgets/TopReadWidget.swift

```diff
@@ -125,49 +125,58 @@ final class TopReadData {
 // MARK: - Model
 
 struct TopReadEntry: TimelineEntry {
-	struct RankedElement: Identifiable {
-		var id: String = UUID().uuidString
-
-		let title: String
-		let description: String
-		var articleURL: URL? = nil
-		var image: UIImage? = nil
-		var thumbnailURL: URL? = nil
-		let viewCounts: [NSNumber]
-	}
-
-	let date: Date // for Timeline Entry
-	var rankedElements: [RankedElement] = Array(repeating: RankedElement.init(title: "–", description: "–", image: nil, viewCounts: [.init(floatLiteral: 0)]), count: 4)
-	var groupURL: URL? = nil
-	var contentLayoutDirection: LayoutDirection = .leftToRight
+    struct RankedElement: Identifiable {
+        var id: String = UUID().uuidString
+
+        let title: String
+        let description: String
+        var articleURL: URL? = nil
+        var image: UIImage? = nil
+        var thumbnailURL: URL? = nil
+        let viewCounts: [NSNumber]
+    }
+
+    let date: Date // for Timeline Entry
+    var rankedElements: [RankedElement] = Array(repeating: RankedElement.init(title: "–", description: "–", image: nil, viewCounts: [.init(floatLiteral: 0)]), count: 4)
+    var groupURL: URL? = nil
+    var contentLayoutDirection: LayoutDirection = .leftToRight
 }
 
 // MARK: - TimelineProvider
 
 struct TopReadProvider: TimelineProvider {
 
-	// MARK: Nested Types
+    // MARK: Nested Types
 
-	public typealias Entry = TopReadEntry
+    public typealias Entry = TopReadEntry
 
-	// MARK: Properties
+    // MARK: Properties
 
-	private let dataStore = TopReadData.shared
+    private let dataStore = TopReadData.shared
 
-	// MARK: TimelineProvider
+    // MARK: TimelineProvider
 
-	func placeholder(in: Context) -> TopReadEntry {
-		return dataStore.placeholder
-	}
+    func placeholder(in: Context) -> TopReadEntry {
+        return dataStore.placeholder
+    }
 
-	func getTimeline(in context: Context, completion: @escaping (Timeline<TopReadEntry>) -> Void) {
-		dataStore.fetchLatestAvailableTopRead { entry in
-			let timeline = Timeline(entries: [entry], policy: .atEnd)
-			completion(timeline)
-		}
-	}
+    func getTimeline(in context: Context, completion: @escaping (Timeline<TopReadEntry>) -> Void) {
+        dataStore.fetchLatestAvailableTopRead { entry in
+            let timeline: Timeline<TopReadEntry>
+            let isError = (entry.groupURL == nil)
+            if !isError {
+                timeline = Timeline(entries: [entry], policy: .atEnd)
```

Comment:
We _could_ change this to match the after midnight policy, but let's leave it. `.atEnd` is the default policy, in use it currently feels like it has the behavior characteristics we desire, and I'm curious if in production it exposes any kind of difference in behavior from the other widgets.